Icon/Icon.module.less

```diff
@@ -1,8 +1,8 @@
 // Icon.module.less
 //
 @import "../styles/mixins.less";
-@import "../styles/variables.less";
 @import "../styles/skin.less";
+@import (css) url("https://fonts.googleapis.com/icon?family=Material+Icons");
```

Comment:
Why did choose to include this import here rather than in fonts.less with the others?

Reply:
This seemed like an interesting optimization, considering this is pulling an external resource. It would only be imported if `Icon` is used. It could be moved into fonts if you think that would be better.

Reply:
If you feel this is how themes should be architected, I have no objection to it here. I asked because it was not what I expected to see.

Reply:
While considering whether this should move into the fonts file or not, I looked into the content of the imported URL (https://fonts.googleapis.com/icon?family=Material+Icons+Round) and it includes a class name, (which we're referencing in the component). Our `fonts.less` file has historically been *just* `@font-face` rules, establishing the fonts, with no classes, as classes are defined elsewhere. This fact adds justification for why this is imported here rather than in a fonts file, which does import other Google font(s). Those, however, follow those `@font-face`-only rules.